core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -281,6 +281,12 @@ class UserService(
     if (roles.nonEmpty) Some(mapCloudPlatformAndPolicies(p, billingProfile, roles, workspaceManagerDAO)) else None
   )
 
+  def getBillingProjectById(id: UUID): Future[Option[RawlsBillingProjectResponse]] =
+    billingRepository.getBillingProjectById(id).flatMap {
+      case Some(billingProject) => getBillingProject(billingProject.projectName)
+      case None                 => Future.successful(None)
```

Comment:
Shouldn't this be a 404?  Or is that handled further up the chain?  Or is this how it's done elsewhere in the code?

Reply:
That is handled further up the chain